test/e2e/service.go

```diff
@@ -932,11 +964,16 @@ func waitForLoadBalancerIngress(c *client.Client, serviceName, namespace string)
 	return service, fmt.Errorf("service %s in namespace %s doesn't have a LoadBalancer ingress point after %.2f seconds", serviceName, namespace, timeout.Seconds())
 }
 
-func waitForLoadBalancerDestroy(c *client.Client, serviceName, namespace string) (*api.Service, error) {
+func waitForLoadBalancerDestroy(c *client.Client, serviceIP, servicePort, serviceName, namespace string) (*api.Service, error) {
 	// TODO: once support ticket 21807001 is resolved, reduce this timeout back to something reasonable
 	// TODO: this should actually test that the LB was released at the cloud provider
 	const timeout = 10 * time.Minute
 	var service *api.Service
+	defer func() {
+		if err := EnsureResourcesDeleted(serviceIP, servicePort); err != nil {
+			Logf("Failed to delete cloud resources for service: %s %s", serviceIP, servicePort)
```

Comment:
we should print the `err`.

notably, this failed on Jenkins, but it's not clear why:
```
11:22:04 STEP: changing service mutability-service-test back to type=ClusterIP
11:22:04 STEP: waiting up to 10m0s for service mutability-service-test in namespace e2e-tests-services-l20si to have no LoadBalancer ingress points
11:22:04 Jan 27 19:21:59.100: INFO: Failed to delete cloud resources for service: 104.197.244.14 80
```